### ISSUE TITLE
Use of image from --allow-enhancement option in OCR-D workflow

### ISSUE THREAD

**Issue by sjscotti** [qurator-spk/eynollah#81]
Hi
A question.  Is the `--allow-enhancement` option resultant image intended to be used in an OCR-D workflow?  In some cases, I believe resultant image has dimensions that differ from the dimensions of the input image.  So if it is intended to be used in OCR-D, what is the suggested workflow (and the commands & parameters) that preserves the overlaying of the ocr results with the image?
Thanks!

**Comment by kba** [comment]
> A question. Is the --allow-enhancement option resultant image intended to be used in an OCR-D workflow?

No, we currently do not use the intermediate results of eynollah, only the resulting PAGE-XML with the assumption that the original image (dimensions) remains unchanged.

**Comment by sjscotti** [comment]
Do you think that the enhancement approaches used in `eynollah` should be replicated outside of it to be used in an OCR workflow, or do you think the OCR-D workflow enhancement steps produce a better quality starting image to do OCR on?

**Comment by vahidrezanezhad** [comment]
> Do you think that the enhancement approaches used in `eynollah` should be replicated outside of it to be used in an OCR workflow, or do you think the OCR-D workflow enhancement steps produce a better quality starting image to do OCR on?

The impact of enhanced image on OCR is not assessed yet. But I think for documents with low quality the enhanced images may improve  OCR performance.
